test/test_assemble.jl

```diff
@@ -139,11 +139,17 @@ end
     as = start_assemble(Symmetric(K))
     errr = ErrorException("some row indices were not found")
```

Comment:
```suggestion
    errr(i,j) = ErrorException(Ferrite._missing_sparsity_pattern_message(i,j))
```
And use below?

Alternatively, remove this line as not used.